src/controllers/userTransform.controller.ts

```diff
@@ -0,0 +1,78 @@
+import { Context } from "koa";
+import { MiscService } from "../services/misc.service";
+import { ProcessorRequest, UserTransfromServiceResponse } from "../types/index";
+import { compileUserLibrary } from "../util/ivmFactory";
+import UserTransformService from "../services/userTransform/userTransform.service";
+import logger from "../logger";
+import { setupUserTransformHandler } from "../util/customTransformer";
+
+export default class UserTransformController {
+  public static async userTransform(ctx: Context) {
+    let requestMetadata = MiscService.getRequestMetadata(ctx);
+    const events = ctx.request.body as ProcessorRequest[];
+    const processedRespone: UserTransfromServiceResponse = await UserTransformService.userTransformRoutine(
```

Comment:
UserTransformService.userTransformRoutine -> UserTransformService.transform (function should be verb/action)